tools/automator/automator.sh

```diff
@@ -280,6 +276,18 @@ merge() {
   fi
 }
 
+validate_changes_exist_in_prior_pr() {
+  if [ -n "$git_exclude" ]; then
+    pr="$(curl -sSfLH "Authorization: token $token" -H "Accept: application/vnd.github.groot-preview+json" "https://api.github.com/repos/$user/$repo/commits/$sha/pulls" | jq ".[0].number")"
+    export GITHUB_TOKEN="$token"
+    changes=$(gh pr view "$pr" --repo "$user"/"$repo" --json files | jq -r '.files[].path' | grep -cvE '$git_exclude')
```

Comment:
Why do we need to look at the PR instead of the commit locally? Aren't they the same?

Reply:
If we always expect the PR to be squashed, then we should be able to look at the last commit. I recall some issues with `git diff-tree` in the past.  I can try just using `git show --name-only`.

Reply:
@howardjohn Changed implementation to use files for the most recent commit. Format it name only with a one-liner as the header removed with the sed. Then the grep will remove the specified lines. If any changes are left, the job continues. Else ends.